codility_training/lessons.lesson09.MaximumSliceProblem.MaxSliceSum/sangmandu.py

```diff
@@ -0,0 +1,22 @@
+# you can write to stdout for debugging purposes, e.g.
+# print("this is a debug message")
+
+def solution(A):
+    # write your code in Python 3.6
+    pass
+    size = len(A)
+    L = [0] * size
+    mx = 0
+
+    if sorted(A)[-1] < 0:
+        return sorted(A)[-1]

```

Comment:
```suggestion
    sortedA = sorted(A);
    if sortedA[-1] < 0:
        return sortedA[-1]
```
연산된 값을 저장해놓는 방식으로 작성해주세요!

Reply:
2번정도 반복할지라도, 변수하나 더 선언하는 메모리를 먹는게 나은 건가요??

Reply:
단순한 반복이 아니라 sorted 같은 연산이라면 사용하는게 낫죠!

Reply:
감사합니다!